src/Views/Table.elm

```diff
@@ -1,22 +1,30 @@
 module Views.Table exposing (view)
 
+import Date exposing (Date)
 import Date.Extra as DateE
 import Html exposing (Html, text, div, img)
 import Material.Table as Table
 import Model exposing (ExpenseItem)
 import Msg exposing (Msg)
+import FormatNumber
+import FormatNumber.Locales
 
+formatCurrency : Float -> String
+formatCurrency =
+  FormatNumber.format FormatNumber.Locales.usLocale
+
```

Comment:
I totally missed this before merge this diff. Can you include elm-format in your toolchain? That takes care of applying appropriate spaces/format automatically.